test/extensions/filters/http/aws_request_signing/config_test.cc

```diff
@@ -21,6 +21,9 @@ TEST(AwsRequestSigningFilterConfigTest, SimpleConfig) {
   const std::string yaml = R"EOF(
 service_name: s3
 region: us-west-2
+match_excluded_headers:
+  prefix: [x-envoy]
+  exact: [foo]
```

Comment:
As I mentioned earlier in the `proto` file, let's add a bit of details as to how a user can specify a header and what sort of matcher we support.